stockroom/storages/param.py

```diff
@@ -0,0 +1,27 @@
+import numbers
+from ..repository import StockRepository
+from .. import parser
+
+
+class ParamStore:
+    def __init__(self, write):
+        self._write = write
+        self._repo = StockRepository()
+
+    def save(self, key, value):
+        if not isinstance(value, numbers.Real):
+            raise TypeError("ParamStore accepts only numbers as input")
+        co = self._repo.checkout(write=True)
```

Comment:
use a context manager. if any error occurs below then the checkout will not be closed.